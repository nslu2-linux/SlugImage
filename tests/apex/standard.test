#!/bin/sh
image=/tmp/slugimage.$$
../slugimage -d -p -o $image -L files/128KiB-16 -k files/1024KiB-32 -r Flashdisk:files/6528KiB
hexdump -C $image > $1
rm -rf $image
