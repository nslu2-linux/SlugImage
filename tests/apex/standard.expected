Ran out of flash space in <Flashdisk> - 0x20000 bytes too large.
